app/views/foods/index.html.erb

```diff
@@ -0,0 +1,8 @@
+Total number of items: <%= @foods.count %><br><br>
```

Comment:
I see why you created all these food objects now. But, there is a `totalHits` key returned from the API  call that we can extract this information from in a more efficient manner.